documentation/configuration-utils/_iam.config.json

```
@@ -0,0 +1,86 @@
+{
+  "acl.enabled": {
+    "default": "true",
+    "description": "Enables/disables Identity and Access Management."
+  },
+  "acl.admin.user.enabled": {
+    "default": "true",
+    "description": "Enables/disables the built-in admin user."
+  },
+  "acl.admin.user": {
+    "default": "admin",
+    "description": "Name of the built-in admin user."
+  },
+  "acl.admin.password": {
+    "default": "quest",
+    "description": "The password of the built-in admin user."
+  },
+  "acl.basic.auth.realm.enabled": {
+    "default": "false",
+    "description": "When enabled the browser's basic auth popup window is used instead of the Web Console's login screen. Only present for backwards compatibility."
+  },
+  "acl.entity.name.max.length": {
+    "default": "255",
+    "description": "Maximum length of user, group and service account names."
+  },
+  "acl.password.hash.iteration.count": {
+    "default": "100000",
+    "description": "QuestDB Enterprise never stores passwords in plain text, it stores password hashes only. This is the number of hash iterations used in password hashing. Higher means safer, almost never should be changed."
+  },
+  "acl.rest.token.refresh.threshold": {
+    "default": "10",
+    "description": "When a REST token is created in REFRESH mode, its TTL is extended on every successful authentication, unless the last successful authentication was within this threshold. This setting removes unnecessary overhead of continuously refreshing REST tokens if they are used often."
```

```suggestion
  "acl.rest.token.refresh.threshold": {
    "default": "10",
    "description": "When a REST token is created in REFRESH mode, its TTL is extended on every successful authentication, unless the last successful authentication was within this threshold. This setting removes unnecessary overhead of continuously refreshing REST tokens if they are used often."
```

@glasstiger Can we specify which unit of time is 10?

---

it is in seconds.
pushed a change to clarify it.